### PR TITLE
chore: upgrade underscore to patch CVE-2021-23358

### PR DESCRIPTION
**Pull request by crgwbr**

### CODE REVIEW

**Comment by crgwbr** [comment]
Ping @yokuze

File: .travis.yml [hunk 1]
```diff
@@ -2,15 +2,13 @@ language: node_js
...
    - "lts/*" # Latest LTS version
-   - "10"
-   - "8"
-   - "8.10"
-   - "6"
+   - "14"
+   - "12"
 
-before_install: if [[ `npm -v` != 6* ]]; then npm i -g npm@6.4.1; fi
+before_install: if [[ `npm -v` != 6* ]]; then npm i -g npm@6.14.12; fi
```
Comment (yokuze) [inline .travis.yml:8]: @jthomerson some of our projects have this if statement and some do not (i.e. they run `npm i -g npm@6.14.12` [every time](https://github.com/silvermine/eslint-plugin-silvermine/blob/master/.travis.yml#L8)) . Do you recall if it's still needed? I can't recall why we do it one way over the other.

**Review by jthomerson — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
   "description": "Replacement for _.template (underscore or lodash) without unsafe evals.",
   "main": "src/index.js",
   "scripts": {
-    "test": "check-node-version --package --print && ./node_modules/.bin/istanbul cover --root src --include-all-sources ./node_modules/.bin/_mocha -- -r ./tests/setup/before.js -R spec 'tests/**/*.test.js'"
+    "test": "check-node-version --package --print && ./node_modules/.bin/istanbul cover --root src --include-all-sources ./node_modules/.bin/_mocha -- -r ./tests/setup/before.js -R spec 'tests/**/*.test.js'",
```
Comment (yokuze) [inline package.json:7]: I had to upgrade from Istanbul to NYC to get rid of some of the old Underscore deps in this PR: https://github.com/silvermine/eslint-plugin-silvermine/pull/55/files

Can we do the same here? Sorry, I should've dropped a note to you about it.

File: package.json [hunk 3]
```diff
@@ -4,7 +4,8 @@
   "description": "Replacement for _.template (underscore or lodash) without unsafe evals.",
   "main": "src/index.js",
   "scripts": {
```
Comment (yokuze) [inline package.json:6]: Another thing you didn't know about (my fault again): Could we add commitlint to this project? All it takes is to add it as a devDependency, then add an npm script like this: https://github.com/silvermine/toolbox/blob/master/package.json#L11

The hash there is generally the git short hash of the commit that adds commitlint.

**Review by yokuze — CHANGES_REQUESTED** [review]
Could we also add an .nvmrc file: https://github.com/silvermine/eslint-config-silvermine/blob/master/.nvmrc ?

**Pull request merged**